nfqueue.go

```diff
@@ -307,23 +312,28 @@ func (nfqueue *Nfqueue) setVerdict(id uint32, verdict int, batch bool, attribute
 }
 
 func (nfqueue *Nfqueue) socketCallback(ctx context.Context, fn HookFunc, errfn ErrorFunc, seq uint32) {
+	defer nfqueue.wg.Done()
```

Comment:
the `nfqueue.wg.Add(1)` for this `.Done()` call is missing.

Reply:
Oops, should be fixed now